tools/cvpdeployment/SampleClaimsProvider/SampleClaimsProvider/ClaimsProviderStore.cs

```diff
@@ -0,0 +1,153 @@
+﻿namespace SampleClaimsProvider
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using System.Collections.Generic;
+    using Microsoft.Azure.Documents;
+    using Microsoft.Azure.Documents.Client;
+    using Microsoft.Azure.Documents.Linq;
+    using Microsoft.Extensions.Logging;
+
+    /// <summary>
+    /// Provides a simplified interface for interacting with a cosmos db DocumentClient as a claims provider store.
+    /// No behavior defined here is required, you may implement it all as you see fit given that the responses provided to CVP are structured appropriately.
+    /// This is merely a sample implementation of a basic claims provider store.
+    /// </summary>
+    public class ClaimsProviderStore
```

Comment:
This class feels like it would be better as a static class, but you don't have to change it.

Reply:
I think I'll leave it as is